packages/isorropia/src/epetra/Isorropia_EpetraZoltanLib.cpp

```diff
@@ -179,9 +179,9 @@ ZoltanLibClass::ZoltanLibClass(Teuchos::RCP<const Epetra_BlockMap> input_map,
 int ZoltanLibClass::precompute()
 {
   std::string str1("Isorropia::ZoltanLibClass::precompute ");
-  MPI_Comm mpicomm = MPI_COMM_WORLD;
+  MPI_Comm mpicomm = MPI_Comm_Default();
```

Comment:
We should add the `Zoltan` prefix to this getter (or put in inside namespace) to avoid name collision.

Reply:
Ok. Fixed